bitnami/discourse/README.md

```diff
@@ -8,6 +8,8 @@ Discourse is an open source discussion platform with built-in moderation and gov
 
 Trademarks: This software listing is packaged by Bitnami. The respective trademarks mentioned in the offering are owned by the respective companies, and use of them does not imply any affiliation or endorsement.
 
+Looking to use Discourse## TL;DRreg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

Comment:
```suggestion
Looking to use Discourse&reg; in production? Try [VMware Application Catalog](https://bitnami.com/enterprise), the enterprise edition of Bitnami Application Catalog.
```

Reply:
This suggestion is duplicated. I will mark this one as resolved.

BTW, thanks for the reviews!